cmd/ethrex/ethrex.rs

```diff
@@ -114,6 +114,13 @@ async fn main() {
         bootnodes = networks::SEPOLIA_BOOTNODES.to_vec();
     }
 
+    if network == "mekong" {
+        warn!("Using mekong presets, bootnodes field will be ignored");
```

Comment:
instead of ignoring the bootnodes field, shouldn't we append it? Thoughts?

Reply:
#1866